fe/fe-core/src/main/java/com/starrocks/load/DeleteHandler.java

```diff
@@ -744,6 +744,10 @@ public void replayDelete(DeleteInfo deleteInfo, Catalog catalog) {
         if (deleteInfo == null) {
             return;
         }
+        if ((System.currentTimeMillis() - deleteInfo.getCreateTimeMs()) / 1000 > Config.label_keep_max_second) {
+            LOG.info("delete info outdated, create time: {}, ignore", deleteInfo.getCreateTimeMs());
+            return;
```

Comment:
should updateTableDeleteInfo before return

Reply:
done